code/game/objects/items/storage/belt.dm

```diff
@@ -414,7 +414,7 @@
 	desc= "A small, lightweight pouch that can be clipped onto Armat Systems M3 Pattern armor or your belt to provide additional storage."
 	storage_slots = 3
 	w_class = WEIGHT_CLASS_BULKY
-	cant_hold = list(/obj/item/ammo_magazine/rifle/standard_lmg)
+	cant_hold = list(/obj/item/ammo_magazine/standard_lmg)
```

Comment:
Is this intended?

Reply:
yes,the g8 pouch should't be able to hold it I guess